spec/components/search_spec.rb

```diff
@@ -1921,4 +1921,37 @@ def indexed_post(*args)
       expect(Search.new("advanced order:chars").execute.posts).to eq([post0, post1])
     end
   end
+
+  context 'search all but topics/posts' do
```

Comment:
```suggestion
  context 'exclude_topics filter' do
```

I think this is much clearer for what we're testing for.